src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -252,3 +253,26 @@ def multiclass_nms(
     }
 
     return _get_node_factory_opset9().create("MulticlassNms", inputs, attributes)
+
+
+def grid_sample(
+        data: NodeInput,
+        grid: NodeInput,
+        attributes: dict,
+        name: Optional[str] = None
+) -> Node:
+    """Return a node which performs GridSample operation.
+
+    :param data: Tensor with data.
+    :param axes: Tensor with grid values (normalized input coordinates).
+    :param attributes: A dictionary containing GridSample's attributes.
+    :return: A new GridSample node.
+    """
+    requirements = [
+        ("align_corners", False, np.bool_, None),
+        ("mode", False, np.str_, None),
+        ("padding_mode", False, np.str_, None),
+    ]
+
+    check_valid_attributes("GridSample", attributes, requirements)
```

Comment:
Wouldn't be easier to add this arguments to `grid_sample` function and add checks for typing in signature? Overall, all attributes errors will be thrown in C++, won't they?